modules/gapi/perf/render/gapi_render_perf_tests_inl.hpp

```diff
@@ -0,0 +1,1144 @@
+    cv::gapi::wip::draw::render(gapi_mat, prims);
+
+    cv::GMat in;
+    cv::GArray<cv::gapi::wip::draw::Prim> arr;
+
+    cv::GComputation comp(cv::GIn(in, arr),
+                          cv::GOut(cv::gapi::wip::draw::render3ch(in, arr)));
+
+    // Warm-up graph engine:
+    comp.apply(gin(gapi_mat, prims), gout(gapi_mat));
+
+    TEST_CYCLE()
+    {
+        comp.apply(gin(gapi_mat, prims), gout(gapi_mat));
+    }
+
+    // OpenCV code //////////////////////////////////////////////////////////////
+    {
+        blendImageRef(ref_mat, org, img, alpha);
+    }
+
+    // Comparison //////////////////////////////////////////////////////////////
+    {
+        EXPECT_EQ(0, cv::norm(gapi_mat, ref_mat));
+    }
+
+    SANITY_CHECK_NOTHING();
+}
+
+PERF_TEST_P_(RenderTestImages, RenderImagesPerformanceNV12OCVTest)
+{
+    cv::Rect rect;
+    cv::Scalar color;
+    double  transparency;
+    MatType type;
+
+    cv::Size sz;
+    std::tie(sz, rect, color, transparency, type) = GetParam();
+
+    cv::Mat y_ref_mat, uv_ref_mat;
+
+    cv::Mat y_in_gapi_mat (sz, CV_8UC1), uv_in_gapi_mat (sz / 2, CV_8UC2),
+            y_out_gapi_mat(sz, CV_8UC1), uv_out_gapi_mat(sz / 2, CV_8UC2);
+
+    create_rand_mats(sz,     CV_8UC1, y_ref_mat  , y_in_gapi_mat);
+    create_rand_mats(sz / 2, CV_8UC2, uv_ref_mat , uv_in_gapi_mat);
+
+    cv::Mat img(rect.size(), CV_8UC3, color);
+    cv::Mat alpha(rect.size(), CV_32FC1, transparency);
+    auto tl = rect.tl();
+    cv::Point org = {tl.x, tl.y + rect.size().height};
+
+    // G-API code //////////////////////////////////////////////////////////////
+    cv::gapi::wip::draw::Prims prims;
+    prims.emplace_back(cv::gapi::wip::draw::Image{org, img, alpha});
+
+    cv::GMat y_in, uv_in, y_out, uv_out;
+    cv::GArray<cv::gapi::wip::draw::Prim> arr;
+    std::tie(y_out, uv_out) = cv::gapi::wip::draw::renderNV12(y_in, uv_in, arr);
+
+    cv::GComputation comp(cv::GIn(y_in, uv_in, arr), cv::GOut(y_out, uv_out));
+
+    // Warm-up graph engine:
+    comp.apply(cv::gin(y_in_gapi_mat, uv_in_gapi_mat, prims),
+               cv::gout(y_out_gapi_mat, uv_out_gapi_mat));
+
+    TEST_CYCLE()
+    {
+        comp.apply(cv::gin(y_in_gapi_mat, uv_in_gapi_mat, prims),
+                   cv::gout(y_out_gapi_mat, uv_out_gapi_mat));
+    }
+
+    // OpenCV code //////////////////////////////////////////////////////////////
+    {
+        // NV12 -> YUV
+        cv::Mat yuv;
+        cv::gapi::wip::draw::cvtNV12ToYUV(y_ref_mat, uv_ref_mat, yuv);
+
+        cv::Mat yuv_img;
+        cv::cvtColor(img, yuv_img, cv::COLOR_BGR2YUV);
+        blendImageRef(yuv, org, yuv_img, alpha);
+
+        // YUV -> NV12
+        cv::gapi::wip::draw::cvtYUVToNV12(yuv, y_ref_mat, uv_ref_mat);
+    }
+
+    // Comparison //////////////////////////////////////////////////////////////
+    {
+        EXPECT_EQ(0, cv::norm(y_out_gapi_mat,  y_ref_mat));
+        EXPECT_EQ(0, cv::norm(uv_out_gapi_mat, uv_ref_mat));
+    }
```

Comment:
It is bad idea to check accuracy in performance tests.
Minimal checks should be applied, like result is not empty and/or is not black(zeroed) completely.

Reply:
So, we can remove all reference code from here, because we check accuracy via unit tests. 
@dmatveev What do you think ?

Reply:
Actually we have followed @elatkin's vast experience in this question. We need to make sure the result is still valid when optimizing our code

Reply:
So after a long-long discussion we decided to go with basic checks only.
According to @TolyaTalamanov , our different (private) OSD backends can never converge to the same accuracy, so in unit tests we test those separately (unlike we do with the rest of our regular kernels).

In order to reuse the performance tests for OSD, we relax this here now.

Reply:
I was sure that we've removed those checks !

@aDanPin Can u comment it ?

Reply:
May "vast experience" includes the cases when people used to test and report great performance of their software, while the software produced completely wrong results.  

What if your software (the backend you test) gets completely broken, say, does not start at all? Performance would look great, but the output would be completely useless.